tests/test_gf_coordinate_search.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
+'''Test WGC (geometry finder) coordinate search calculation.'''
+
+import pytest
+
+from webgeocalc import GFCoordinateSearch
+from webgeocalc.errors import CalculationInvalidAttr, CalculationInvalidValue
+
+from webgeocalc.vars import ESA_URL, JPL_URL
+
+
+@pytest.fixture
+def kernels():
+    '''Generic kernel set.'''
+    return 1
+
+@pytest.fixture
+def intervals():
+    '''Input time interval'''
+    return ["1986-08-22", "1986-08-23"]
+
+@pytest.fixture
+def reference_frame():
+    '''Input reference frame.'''
+    return 'EARTH_FIXED'
+
+@pytest.fixture
+def observer():
+    '''Input observer.'''
+    return 'EARTH'
+
+@pytest.fixture
+def target():
+    '''Input observer.'''
+    return 'JUPITER'
```

Comment:
The example [in the docs](https://wgc2.jpl.nasa.gov:8443/webgeocalc/documents/api-info.html#GF_COORDINATE_SEARCH) is for Enceldadus seen by Cassini in 2012-10-19 between 7 am and 9 am.

Reply:
I used the Cassini Enceladus data